src/components/rangeslider/defaults.js

```diff
@@ -60,15 +60,18 @@ module.exports = function handleDefaults(layoutIn, layoutOut, axName) {
             var rangeContainerIn = containerIn[yName] || {};
             var rangeContainerOut = containerOut[yName] = {};
 
+            var yAxOut = layoutOut[yName];
+
             var rangemodeDflt;
-            if(rangeContainerIn.range && layoutOut[yName].isValidRange(rangeContainerIn.range)) {
+            if(rangeContainerIn.range && yAxOut.isValidRange(rangeContainerIn.range)) {
                 rangemodeDflt = 'fixed';
             }
 
             var rangeMode = coerceRange(rangeContainerIn, rangeContainerOut, 'rangemode', rangemodeDflt);
             if(rangeMode !== 'match') {
-                coerceRange(rangeContainerIn, rangeContainerOut, 'range', layoutOut[yName].range.slice());
+                coerceRange(rangeContainerIn, rangeContainerOut, 'range', yAxOut.range.slice());
```

Comment:
Ok. Looks like `rangeslider.yaxis?.range` is still getting coerced when `rangemode: 'auto'` though this appears standard for subplot `xaxis` and `yaxis` `range` under `autorange: true`, so this sounds fine :ok_hand: . 

That said, it would be nice in the future to make:

```js
Plotly.validate([], {
  xaxis: {
    autorange: true,
    range: [0, 1]
  }
})
```

spit out an `unused` log message for `xaxis.range`